server/routerlicious/packages/lambdas/src/alfred/index.ts

```diff
@@ -221,12 +217,12 @@ export function configureWebSocketServices(
             try {
                 await tenantManager.verifyToken(claims.tenantId, token);
             } catch (err) {
-                // eslint-disable-next-line prefer-promise-reject-errors
-                return Promise.reject({
-                    // if we don't understand the error, be lenient and allow retry
-                    code: err?.response?.status ?? 401,
-                    message: err?.response?.data ?? "Invalid token",
-                });
+                if (typeof (err as NetworkError | undefined)?.code === "number" &&
+                    typeof (err as NetworkError | undefined)?.message === "string") {
+                    throw (err as NetworkError);
+                }
+                // If we don't understand the error, be lenient and allow retry.
+                throw new NetworkError(401, "Invalid token");
```

Comment:
Similar comment as above about 403 when the token is missing - What does `tenantManager.verifyToken` do?  Is it validating access or just that the token is meaningful at all?  If it's checking access I'd expect this to be 403.

Reply:
`tenantManager.verifyToken` makes an internal HTTP request to Riddler service where the token is validated for both correct secret used for signing and that it is not expired. The above check for error as a NetworkError takes that into account. Riddler will throw a 403 if the JWT is signed with an incorrect key, or it will throw a 401 if the token is expired. The reason for this 401 is that we want to give the client the opportunity to generate a new token and retry if there is some other reason for failure. Perhaps it should be a 500 though

Reply:
Updated so that unrecognized errors throw a 500 (Internal Server Error) which is retriable but does not indicate invalid credentials.